sdk/src/main/java/com/sygic/travel/sdk/Sdk.kt

```diff
@@ -12,8 +12,17 @@ import com.sygic.travel.sdk.tours.facade.ToursFacade
 /**
  * Provides public methods for requesting API.
  */
-class Sdk(clientId: String, xApiKey: String, context: Context) {
-	private var kodein: Kodein = KodeinSetup.setupKodein(clientId, xApiKey, context)
+class Sdk(
+	clientId: String,
+	xApiKey: String,
+	context: Context,
+	urlConfig: UrlConfig
```

Comment:
isn't possible to do something like urlConfig = UrlConfig {}?

Reply:
No.

Reply:
Neither `urlConfig = object : UrlConfig {}`

Reply:
you should try harder... `urlConfig: UrlConfig = object : UrlConfig {}` seems working ;)

Reply:
Some Kotlin magic :D